src/main/kotlin/com/mineinabyss/looty/features/recipes/AllowInVanillaRecipe.kt

```diff
@@ -0,0 +1,12 @@
+package com.mineinabyss.looty.features.recipes
+
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+/**
+ * A component to indicate that an ItemStack should be allowed in vanilla crafting recipes.
+ * Meaning if a GearyItem has a base-material of PAPER and this component, it can be used to craft books.
+ */
+@Serializable
+@SerialName("looty:allow_in_vanilla_recipes")
+class AllowInVanillaRecipes
```

Comment:
Probably best to change this component to deny in vanilla recipes, since we don't want to change vanilla behaviour by default. We might have prefabs for vanilla items in the future to let us customize their behaviour for instance.